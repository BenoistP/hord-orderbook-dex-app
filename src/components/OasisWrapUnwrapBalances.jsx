import React, { PureComponent } from 'react';
import { PropTypes } from 'prop-types';
// import ImmutablePropTypes from 'react-immutable-proptypes';

import styles from './OasisWrapUnwrapBalances.scss';


const propTypes = PropTypes && {
};
const defaultProps = {};


class OasisWrapUnwrapBalances extends PureComponent {
  render() {
    return (
      <div className={styles.base}>
      </div>
    );
  }
}

OasisWrapUnwrapBalances.displayName = 'OasisWrapUnwrapBalances';
OasisWrapUnwrapBalances.propTypes = propTypes;
OasisWrapUnwrapBalances.defaultProps = defaultProps;
export default OasisWrapUnwrapBalances;
