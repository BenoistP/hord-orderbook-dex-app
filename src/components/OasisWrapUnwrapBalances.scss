
.table {


  th:first-child,
  td:first-child {
    flex-basis: 100px;
    text-align: right;
  }

  th:nth-child(2),
  td:nth-child(2) {
    flex-basis: 150px;
    text-align: right;
  }

  th:nth-child(3),
  td:nth-child(3) {
    flex-basis: 150px;
    text-align: right;
  }

}
