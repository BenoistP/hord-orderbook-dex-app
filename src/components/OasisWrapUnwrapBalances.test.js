/* global shallow describe it expect */
/* eslint-disable import/first */
import React from 'react';
import { shallow } from 'enzyme';
import OasisWrapUnwrapBalances from './OasisWrapUnwrapBalances';

describe('(Component) OasisWrapUnwrapBalances', () => {
  it('should render', () => {
    const props = {
      children: (
        <span>test</span>
      ),
    };
    const wrapper = shallow(
      <OasisWrapUnwrapBalances {...props}/>,
    );

    expect(wrapper).toMatchSnapshot();
  });
});
