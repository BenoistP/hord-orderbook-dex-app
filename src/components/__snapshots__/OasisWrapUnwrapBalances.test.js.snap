// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`(Component) OasisWrapUnwrapBalances should render 1`] = `<div />`;
